IIS-Sample.yaml

```diff
@@ -36,10 +36,12 @@ spec:
 apiVersion: v1
 kind: Service
 metadata:
-  name: windows-service
+  name: iis-sample
 spec:
   selector:
-    app: windowsservice
+    app.kubernetes.io/name: iis-sample-service
```

Comment:
is `app: iis-sample-service` not working?

Reply:
Is it working! I just checked with the k8 docs and they had the naming as app.kubernetes.io/name: instead of just app: so I made the change!